### PR TITLE
Added videos and individual information

### LINKED ISSUE #309: Run peer testing sessions and fill out report
https://github.com/jpabadir/pace-c/blob/develop/documentation/peer-testing-1/peer-testing-report.md

Notice common vs individual tag: if common, one person in the team will be assigned to it, if individual, *you* have to do it, and so does everyone else. 

Ask for clarification if needed, and make PRs to fill it.

### PR DESCRIPTION
**Pull request by aleksssk**
Aleks peer testing - Added videos and individual information

### CODE REVIEW

File: documentation/peer-testing-1/peer-testing-report.md [hunk 1]
```diff
@@ -27,18 +27,37 @@ In total, \_\_\_ individuals took part in our peer testing sessions. The members
...
+- When signing in, if the user does not enter anything for email and password, it just says "please input something" for both fields, and then a pop-up that says "the email address is badly formatted" however both fields are blank. (login page)
+- From the above point, we should change "please input something" to "please input password" or "please input email", for the red text that pops up at the bottom of the field if the user does not enter an email address in the login page. (login page)
+- If the user enters a incorrect username and password, the website creates a popup that is "wordy" and maybe difficult to understand. (login page)
+- We should make different messages depending on of if the user did not enter any information in login screen (password/email) or if the user entered information but the information is wrong/does not have an account. (login page)
+- From above points, change the login pop up messages depending on what is and is not filled in. Change the red text that popups at the bottom of the field depending on what is and is not filled in. (login page)
```
Comment (katmartel) [inline documentation/peer-testing-1/peer-testing-report.md:36]: From reading these lines, I think they're more or less the same feedback (to change the red error text to be more descriptive, and modify the popups)? I might be misunderstanding though, but if not, can this section be condensed into one or two lines?

File: documentation/peer-testing-1/peer-testing-report.md [hunk 2]
```diff
@@ -27,18 +27,37 @@ In total, \_\_\_ individuals took part in our peer testing sessions. The members
...
+- Make the "organization" field not mandatory. This way, people without an organziation can sign up.
```
Comment (katmartel) [inline documentation/peer-testing-1/peer-testing-report.md]: I would remove this. From speaking with the client, it sounds like she only wants individuals from Organizations to be Mentors. 
```
- Make the "organization" field not mandatory. This way, people without an organziation can sign up.
```

**Review by katmartel — CHANGES_REQUESTED** [review]
Thanks for adding in the information from the peer testing sessions! The participants gave some nice feedback, so that's great. I made two comments that may require some minor changes

File: documentation/peer-testing-1/peer-testing-report.md (hunk 1 above)
Comment (aleksssk) [reply documentation/peer-testing-1/peer-testing-report.md:36]: I might of explained them badly, but they are supposed to be multiple things depending on the fields, like when information is wrong or not filled in then a popup will show, and then the red underneath the box when information is not filled in/wrong. So i was trying to explain that we need to fix both the popup (based on information that is filled but wrong, or information that is not filled in at all) and then we need to fix the read that shows around the box (based on information that is filled but wrong, or information that is not filled in at all). Sorry my explanations are so bad to describe what I mean

File: documentation/peer-testing-1/peer-testing-report.md (hunk 2 above)
Comment (aleksssk) [reply documentation/peer-testing-1/peer-testing-report.md]: Sounds good! removed!

**Review by jpabadir — APPROVED** [review]

File: documentation/peer-testing-1/peer-testing-report.md (hunk 1 above)
Comment (katmartel) [reply documentation/peer-testing-1/peer-testing-report.md:36]: No worries! Thanks for elaborating.

**Review by katmartel — APPROVED** [review]

**Review by wrc9817 — APPROVED** [review]
After addressing their comments, you are good to merge

**Pull request merged**